src/main/scala/nl.knaw.dans.easy.multideposit/actions/ValidatePreconditions.scala

```diff
@@ -36,6 +37,7 @@ class ValidatePreconditions(ldap: Ldap) extends DebugEnhancedLogging {
       _ <- checkSFColumnsIfDepositContainsAVFiles(deposit)
       _ <- checkEitherVideoOrAudio(deposit)
       _ <- checkDepositorUserId(deposit)
+      //_ <- checkBaseRevisionConformsToUUID(deposit)
```

Comment:
Please remove all changes in this class. It turned out you didn't need any additions in here.